tests/integration/helpers/cluster.py

```diff
@@ -1575,6 +1642,10 @@ def add_instance(
         with_nginx=False,
         with_redis=False,
         with_minio=False,
+        # The config is defined in tests/integration/helpers/remote_database_disk.xml
+        # However, some tests cannot use with_remote_database_disk by their configs: e.g using secure keeper
+        # So, we set the default value of with_remote_database_disk to None and try to enable it if possible in DEBUG and ASAN build (i.e. if not explicitly set to false)
+        with_remote_database_disk=None,
```

Comment:
This looks redundant (and also some related code below)

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/83581/commits/fd81d6c302afbfe8c9dd60c35ebca1f5bbc6b87b where I chose only the rabbitmq-related changes